tests/foreman/cli/test_remoteexecution.py

```diff
@@ -252,7 +252,12 @@ def test_positive_run_default_job_template(self):
             'inputs': 'command="ls"',
             'search-query': "name ~ {0}".format(self.client.hostname),
         })
-        self.assertEqual(invocation_command['success'], u'1')
+        self.assertEqual(invocation_command['success'], u'1',
```

Comment:
Thanks @oshtaier, went with the first option as the second calls the output even if assert is true. The example command output (from standalone automation job no 386):

```
E               AssertionError: False is not true : host output: Error initializing command #<Proxy::RemoteExecution::Ssh::Dispatcher::Command:0x007faeb0437040> SocketError getaddrinfo: Name or service not known Exit status: EXCEPTION
```